website/content/en/preview/AWS/user-data.md

```diff
@@ -0,0 +1,129 @@
+---
```

Comment:
I was talking with @chrisnegus and we were thinking it would be better to minimize the number of docs to improve control-f. It's not like our docs are so long that they're unmanagable. What do you think about leaving this in provisioning.md?

Reply:
I agree we could better structure our docs and minimize them, but I'm not convinced about moving all of this back into the main AWS provisioning.md doc. 

There's a lot of info here, and I expect this will continue to grow with Custom AMI support (Custom AMI families, maybe different behavior for Ubuntu / Al2022 in the future) so I think a separate doc is appropriate.